content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -254,18 +257,18 @@ kubectl create configmap game-config-env-file \
        --from-env-file=configure-pod-container/configmap/game-env-file.properties
 ```
 
-would produce the following ConfigMap:
+would produce a ConfigMap. View the ConfigMap:
 
 ```shell
 kubectl get configmap game-config-env-file -o yaml
 ```
 
-where the output is similar to this:
+the output is similar to:
 ```yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  creationTimestamp: 2017-12-27T18:36:28Z
+  creationTimestamp: 2019-12-27T18:36:28Z
```

Comment:
Not sure why use 2019 instead of 2017, while line 133 use 2022.

Reply:
I reckon it'd be fine to bump that `creationTimestamp`.

Reply:
Unless we _have_ to bump this, let's leave it.